src/python/WMComponent/AgentStatusWatcher/DrainStatusPoller.py

```diff
@@ -74,21 +76,26 @@ def updateAgentSpeedDrainConfig(self, thresholdsHit):
         Takes a list of speed drain configuration keys and updates the agent configuration
         """
         updateConfig = False
+        condorPriorityFlag = False
 
         if 'CondorPriority' in thresholdsHit:
             if self.condorAPI.editCondorJobs(
```

Comment:
Do we actually need to check whether the condor_qedit was successful or not? Why don't we just run the edit jobs and set the Enabled flag to True (if it's still False)?